### ISSUE TITLE
Double inputs

### ISSUE THREAD

**Issue by ravinggenius** [boneskull/angular-tags#17]
When adding a new tag, the text I type shows twice. I inspected the DOM in Chrome and noticed that two `<input />`s where present. I believe this issue may be related to #16. Both inputs show and are databound to the model, so when a user types in one, it shows up in both places. Obviously this will be confusing. After typing a tag in either `<input />` and `Enter`, the tag appears to be created as normal, and only one styled tag is show.

![screen shot 2013-10-18 at 11 25 20](https://f.cloud.github.com/assets/21517/1361957/0cf4aa18-3810-11e3-9910-14a0b0363863.png)

![screen shot 2013-10-18 at 12 34 38](https://f.cloud.github.com/assets/21517/1362123/40fa94d2-3813-11e3-9130-7097194513c9.png)

In my view:

``` html
<tags data-options="{addable: true}" typeahead-options="typeaheadOpts" data-model="document.tags"></tags>
```

Inspected input DOM:

``` html
<input ng-if="!srcTags.length" type="text" data-ng-model="inputTag" class="decipher-tags-input ng-valid ng-dirty" />

<input ng-if="srcTags.length" type="text" data-ng-model="inputTag" class="decipher-tags-input ng-valid ng-dirty" data-typeahead="stag as stag.name for stag in srcTags|filter:$viewValue|orderBy:orderBy" data-typeahead-input-formatter="" data-typeahead-loading="" data-typeahead-min-length="" data-typeahead-template-url="bower_components/bootstrap/template/typeahead/typeahead-popup.html" data-typeahead-wait-ms="" data-typeahead-editable="" data-typeahead-on-select="add($item) &amp;&amp; selectArea() &amp;&amp; typeaheadOptions.onSelect()" />
```

**Comment by boneskull** [comment]
Yes, I believe this is some sort of problem related to #16.  Once I confirm we can close this.

**Comment by lefnire** [comment]
Also having this problem on `master`. You can see the issue here ([codepen](http://codepen.io/lefnire/pen/rEGoc?editors=101)), enter "lamb", hit <ENTER> and you get it twice

**Comment by lefnire** [comment]
Sorry, just realized mine's a bit different. If you have `{addable:true}` in `options`, it's  worse: when you hit ENTER that second input tag gets added:

![screen](http://i.gyazo.com/bfd61729b3789f68cb499676fb00c399.png)

Likely due to the same issue though, so I'll keep convo here unless I should create a new ticket.

**Comment by lefnire** [comment]
Seems to do with the `$timeout` [here](https://github.com/boneskull/angular-tags/blob/master/src/tags.js#L71). When I remove it [like so](https://github.com/CirrusCPQ/angular-tags/commit/b1e5c9e1f1bf0715fe3e7af8aa4f952c9c07fb39), it's fixed. I'm obviously missing something that's gonna bite me soon, since that `$timeout` was put there to [patch something](https://github.com/boneskull/angular-tags/blob/master/src/tags.js#L70) going on in typeahead.

**Comment by boneskull** [comment]
I hope to have this corrected in 1.0.0, but I'm still trying to get [grunt-bowerspawn](http://github.com/boneskull/grunt-bowerspawn/) working properly.
